### ISSUE TITLE
Countdown expired

### ISSUE THREAD

**Issue by brianteeman** [brianteeman/haraka#2]
Need to hide the clock if we already reach 0-0-0 0:0:0

**Comment by zero-24** [comment]
We should be able to catch the `finish.countdown` event.
http://hilios.github.io/jQuery.countdown/documentation.html#events

Maybe something like:
```js
$('#counter').countdown('<?php echo $countdownDate; ?>')
.on('finish.countdown', function(event) {
$('#counter').hide();
}
```
Or do i miss something?

**Comment by brianteeman** [comment]
yeah I know how to do it. I just hadn't found the time to do anything. More specifically I haven't deciced what options to have when the clock expires. A message or jus nothing etc

**Comment by brianteeman** [comment]
closed by accident

**Comment by OptikWebCreative** [comment]
maybe at end of countdown disable Haraka, and bring the site / page online
